Source/WebCore/page/FrameView.cpp

```diff
@@ -253,7 +253,7 @@ FrameView::~FrameView()
     
     ASSERT(!m_scrollCorner);
 
-    ASSERT(frame().view() != this || !frame().contentRenderer());
+    ASSERT(m_frame.get().view() != this || !m_frame.get().contentRenderer());
```

Comment:
Ugh, all these `m_frame.get().` in this file should be `m_frame->`.

Reply:
I like m_frame.get(). better because it makes it more clear that nothing can be null, whereas m_frame-> looks like it might be a RefPtr.  I made this suggested change, though.

Reply:
`m_frame.get().` : m_frame could still be a RefPtr.

Reply:
It would need to be m_frame.get()->